tests/test_database_manager.py

```diff
@@ -0,0 +1,150 @@
+
+from . import setup_test
+import unittest
+import database_connector
+import database_manager
+import product_manager
+import helper
+import image_manager
+
+
+class TestDatabaseManager(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.database_connector = database_connector.DatabaseConnector()
+        self.item_manager = database_manager.ItemManager(
+            self.database_connector)
+        self.tag_manager = database_manager.TagManager(
+            self.database_connector)
+
+    def setUp(self):
+        setup_test.create_test_database_setup(self.database_connector)
+
+    def test_add_item(self):
+        # arrange
+        item_id = helper.get_id()
+
+        # act
+        self.item_manager.add_item(item_id, item_id, item_id)
+
+        # assert
+        self.assertTrue(setup_test.assertCount(
+            self.database_connector, 'Items', 1))
+
+    def test_delete_item(self):
+        # arrange
+        item_ids = setup_test.test_items_tags_setup(
+            self.item_manager, self.tag_manager)
+
+        # act
+        self.item_manager.delete_item(item_ids[0])
+
+        # assert
+        self.assertTrue(setup_test.assertCount(
+            self.database_connector, 'Items', 3))
+
+    def test_get_item_details(self):
+        # arrange
+        item_id = helper.get_id()
+        self.item_manager.add_item(item_id, item_id, item_id)
+
+        # act
+        item_details = self.item_manager.get_item_details(item_id)
+
+        # assert
+        self.assertEqual(len(item_details), 3)
+        self.assertIn(item_id, item_details['VectorPath'])
+        self.assertIn(item_id, item_details['PngPath'])
```

Comment:
You could use the `ITEM_ID_1` etc here, as I hinted previously.

Reply:
will update this later on different branch